### ISSUE TITLE
Ability to alter meta viewport

### ISSUE THREAD

**Issue by ekoeryanto** [vuejs/vuepress#845]
## Feature request
Halo, I am working in amp, found `<meta name="viewport" content="width=device-width,initial-scale=1">` can not be overwritten, I hope this can be done in next version, 
thanks

**Comment by meRyanWu** [comment]
It can be overwritten by adding a new meta array in config.js. Then it will appear two `<viewport>` tag in your html `<head>` tag, the latter one will overwrite the previous one.

**Comment by ekoeryanto** [comment]
wait, maybe I miss something in the docs about meta inside config.js

**Comment by ekoeryanto** [comment]
can be resolved with #733

**Comment by b0g3r** [comment]
For any strangers, who try to set viewport: just add it to config.js -> head, e.g.
```
module.exports = (ctx) => ({
  head: [
    ['meta', { name: 'viewport', content: 'width=device-width, initial-scale=1' }],
  ],
...
```
